sql/item_basic.sql

```diff
@@ -19499,15 +19499,15 @@ INSERT INTO `item_basic` VALUES (26178,0,'dimensional_ring_(mea)','dim._ring_(me
 INSERT INTO `item_basic` VALUES (26179,0,'varar_ring','varar_ring',1,2084,25,0,0);
 INSERT INTO `item_basic` VALUES (26180,0,'varar_ring_+1','varar_ring_+1',1,2080,25,0,0);
 INSERT INTO `item_basic` VALUES (26181,0,'chirich_ring','chirich_ring',1,2084,25,0,0);
-INSERT INTO `item_basic` VALUES (26182,0,'chirich_ring_+1','chirich_ring_+1',1,2080,25,0,0);
+INSERT INTO `item_basic` VALUES (26182,0,'chirich_ring_+1','chirich_ring_+1',1,2080,25,0,10587);
 INSERT INTO `item_basic` VALUES (26183,0,'stikini_ring','stikini_ring',1,2084,25,0,0);
-INSERT INTO `item_basic` VALUES (26184,0,'stikini_ring_+1','stikini_ring_+1',1,2080,25,0,0);
+INSERT INTO `item_basic` VALUES (26184,0,'stikini_ring_+1','stikini_ring_+1',1,2080,25,0,12512);
```

Comment:
Even my mythic BLM never got one of these!

Reply:
Nyu is OP